src/Eloquent/Model.php

```diff
@@ -155,7 +155,11 @@ public function getAttribute($key)
         }
 
         // This checks for embedded relation support.
-        if (method_exists($this, $key) && ! method_exists(self::class, $key)) {
+        if (
+            method_exists($this, $key)
+            && ! method_exists(self::class, $key)
+            && ! $this->hasAttributeGetMutator($key)
```

Comment:
Add additional test cases, please

Reply:
I've added a test in the Model/User. If there is anywhere else appropriate or relevant please let me know :)